src/util/updateSitemap.ts

```diff
@@ -268,10 +288,10 @@ const getEventsPage = async (start: Date, page: number) => {
  * @param {string} start
  * @return {Object[]}
  */
-const getEvents = async (start: Date) => {
+const getEvents = async (eventType: 'event' | 'course', start: Date) => {
```

Comment:
You could also fetch all internet based events/courses. Example request: https://api.hel.fi/linkedevents/v1/event?start=now&internet_based=True&&super_event_type=umbrella,none. Now they are not added to sitemap

Reply:
good catch!

Reply:
You can also wait until `all_ongoing=true` parameter is implemented and get all events using just one request. https://helsinkicity.slack.com/archives/C019AV8SKA9/p1608734354007800

Reply:
all_ongoing=true seems to return less events